.travis.yml

```diff
@@ -0,0 +1,59 @@
+sudo: false
+
+language: cpp
+
+os:
+  - linux
+
+compiler:
+  - gcc
+
+env:
+  global:
+    # MPC settings
+    - MPC_ROOT=$TRAVIS_BUILD_DIR/ext/MPC
+    # ACE/TAO settings
+    - ACE_ROOT=$TRAVIS_BUILD_DIR/ext/ACE_TAO/ACE
+    - TAO_ROOT=$TRAVIS_BUILD_DIR/ext/ACE_TAO/TAO
+    - LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ACE_ROOT/lib
+    # OpenDDS settings
+    - DDS_ROOT=$TRAVIS_BUILD_DIR/ext/OpenDDS
+    - LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$DDS_ROOT/lib
+    # LASAGNE settings
+    - DAF_ROOT=$TRAVIS_BUILD_DIR
+    - TAF_ROOT=$DAF_ROOT/TAF
+    - LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$DAF_ROOT/lib
+
+before_install:
+  - mkdir ext
+  - cd ext
+  - git clone --depth 1 git://github.com/DOCGroup/MPC.git
```

Comment:
Would recommend to use the Latest_Micro tag for MPC/ACE_TAO. If you use master you have the risk that everything fails when ACE/TAO master is unstable